From 452d3f554d7e6e018692f9e7befe589e672adc3e Mon Sep 17 00:00:00 2001
From: zhengq <qi.zheng@intel.com>
Date: Wed, 11 Feb 2015 21:39:39 +0800
Subject: [PATCH] Pinctrl: Fix wrong pin definitions in CHV

The pin I2C6_SDA to I2C_NFC_SCL should locate in the
range of [47:52] in the southwest community. The previous
settings [49:54] are wrong.

Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-20518
Tracked-On: https://jira01.devtools.intel.com/browse/AKP-1112
Signed-off-by: zhengq <qi.zheng@intel.com>

Change-Id: I03a6dde0ef15f9165da793cd0aea2e8e03fcb863
---
 drivers/pinctrl/pinctrl-cherryview.c | 17 ++++++++---------
 1 file changed, 8 insertions(+), 9 deletions(-)

diff --git a/drivers/pinctrl/pinctrl-cherryview.c b/drivers/pinctrl/pinctrl-cherryview.c
index ee82f57..8927ed3 100644
--- a/drivers/pinctrl/pinctrl-cherryview.c
+++ b/drivers/pinctrl/pinctrl-cherryview.c
@@ -255,13 +255,12 @@ enum {
 
 	I2C5_SDA = 45,
 	I2C4_SDA = 46,
-
-	I2C6_SDA = 49,
-	I2C5_SCL = 50,
-	I2C_NFC_SDA = 51,
-	I2C4_SCL = 52,
-	I2C6_SCL = 53,
-	I2C_NFC_SCL = 54,
+	I2C6_SDA = 47,
+	I2C5_SCL = 48,
+	I2C_NFC_SDA = 49,
+	I2C4_SCL = 50,
+	I2C6_SCL = 51,
+	I2C_NFC_SCL = 52,
 
 	I2C1_SDA = 60,
 	I2C0_SDA = 61,
@@ -602,14 +601,14 @@ static struct gpio_pad_info southwest_pads_info[] = {
 	/*45~59*/
 	[I2C5_SDA]			= { 3,		0,	-1 },
 	[I2C4_SDA]			= { 3,		1,	-1 },
-	[47]				= { -1,		-1,	-1 },
-	[48]				= { -1,		-1,	-1 },
 	[I2C6_SDA]			= { 3,		2,	-1 },
 	[I2C5_SCL]			= { 3,		3,	-1 },
 	[I2C_NFC_SDA]		= { 3,		4,	-1 },
 	[I2C4_SCL]			= { 3,		5,	-1 },
 	[I2C6_SCL]			= { 3,		6,	-1 },
 	[I2C_NFC_SCL]		= { 3,		7,	-1 },
+	[53]				= { -1,		-1,	-1 },
+	[54]				= { -1,		-1,	-1 },
 	[55]				= { -1,		-1,	-1 },
 	[56]				= { -1,		-1,	-1 },
 	[57]				= { -1,		-1,	-1 },
-- 
1.9.1

